infrastructure/terraform/.terraform.lock.hcl

```diff
@@ -0,0 +1,115 @@
+# This file is maintained automatically by "terraform init".
```

Comment:
Could you remove this file from the PR?

Reply:
@se02035 Should I remove it ? I saw you put it

Reply:
It's best practice to have the lock file in the version control system. To better control terraforms dependency management.

Pls see this https://www.terraform.io/language/files/dependency-lock

@flecoqui @xpouyat

Reply:
@flecoqui Ok to keep the file ?